src/core_modules/capture-core/src/components/D2Form/field/buildField.js

```diff
@@ -210,11 +210,12 @@ const fieldForTypes = {
             props,
         }, metaData);
     },
-    [elementTypes.TIME]: (metaData: MetaDataElement, options: Object) => getBaseTextField(metaData, options),
-    [elementTypes.PERCENTAGE]: (metaData: MetaDataElement, options: Object) => getBaseTextField(metaData, options),
-    [elementTypes.URL]: (metaData: MetaDataElement, options: Object) => getBaseTextField(metaData, options),
+    [elementTypes.TIME]: (metaData: MetaDataElement, options: Object) => getBaseTextField(metaData),
+    [elementTypes.PERCENTAGE]: (metaData: MetaDataElement, options: Object) => getBaseTextField(metaData),
+    [elementTypes.URL]: (metaData: MetaDataElement, options: Object) => getBaseTextField(metaData),
+    [elementTypes.ORGANISATION_UNIT]: (metaData: MetaDataElement, options: Object) => getBaseTextField(metaData),
```

Comment:
You would need to replace the getBaseTextField() with a call that builds your orgunit field. You can look at the BOOL field as an example.